test/test_torch.py

```diff
@@ -13000,6 +13133,21 @@ def test_copy_mem_overlap(self, device):
         self.unary_check_input_output_mem_overlap(
             doubles, sz, lambda input, out: out.copy_(input))
 
+    @torchtest.for_all_device_types()
+    def test_pow_scalar_overloads_mem_overlap(self, device):
```

Comment:
TBH this should probably be a part of `test_binary_op_mem_overlap` and we should test Tensor-scalar overloads for each one of the ops listed there. This doesn't have to be fixed in this pull request, but we should at least have an issue open for it.